src/transformers/models/mctc/tokenization_mctc.py

```diff
@@ -0,0 +1,656 @@
+# coding=utf-8
```

Comment:
think we can actually delete this since the tokenizer should be 1-to-1 the same as Wav2Vec2